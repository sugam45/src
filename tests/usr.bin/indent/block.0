/* $NetBSD: block.0,v 1.1 2021/03/08 20:55:34 rillig Exp $ */
/* $FreeBSD$ */

void
function(void)
{
	if (true) {

	}

	{
		print("block");
	}
}
