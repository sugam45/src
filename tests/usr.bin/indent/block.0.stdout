/* $NetBSD: block.0.stdout,v 1.1 2021/03/08 20:55:34 rillig Exp $ */
/* $FreeBSD$ */

void
function(void)
{
	if (true) {

/* $ FIXME: indent must not merge these braces. */
	} {
/* $ FIXME: the following empty line was not in the input. */

		print("block");
	}
}
